Cargo.toml

```diff
@@ -91,9 +91,9 @@ name = "fish_key_reader"
 path = "src/bin/fish_key_reader.rs"
 
 [features]
-default = ["installable"]
+default = ["embed-data"]
```

Comment:
ah so this is the default for `cargo` commands (making it easy to test) but cmake builds pass `--no-default-features`.